### ISSUE TITLE
mixin error

### ISSUE THREAD

**Issue by UpcraftLP** [AlexIIL/SimplePipes#7]
when setting the `mixn.debug` flag in my dev workspace I get [this error](https://github.com/AlexIIL/SimplePipes/files/3025327/stack_error.log).

**Comment by AlexIIL** [comment]
Yeah I'm not sure why that's happening - as the mixin seems to apply properly for me.
